Completion/X/Command/_kdeconnect-cli

```diff
@@ -0,0 +1,35 @@
+#compdef kdeconnect-cli
+
```

Comment:
The normal naming convention for zsh completion functions is to use the name of the software rather than the command name. So in this case, it should probably be just _kdeconnect
Similarly, we have _subversion for svn (and svn-admin), _perforce for p4, _ansible for ansible-playbook and so on.